mh/pat.py

```diff
@@ -2405,6 +2453,24 @@ def notify_layer_departure(self):
             ntc_data = pati.lp2_string(self.session.capcom_id)
             self.server.layer_broadcast(self.session, PatID4.NtcLayerOut,
                                         ntc_data, 0)
+        if self.session.local_info['circle_id'] is not None:
+            circle = self.session.get_circle()
+            self.notify_circle_leave(session.local_info['circle_id'] + 1, 0)
+
+    def notify_circle_leave(self, circle_index, seq):
+        circle = self.session.get_circle()
+        self.sendNtcCircleLeave(circle, circle_index, seq)
+        if circle.leader == self.session:
+            if circle.departed:
+                new_host = self.session.try_transfer_circle_leadership()
+                if new_host:
+                    # Unknown if NtcCircleHostHandover needs to be sent.
+                    #self.sendNtcCircleHostHandover(circle, new_host[1], new_host[0], seq)
```

Comment:
This (exact) commented line should be removed. 

I wonder where/when this packet should be used then?

Reply:
This packet doesn't seem to have any actual use, I traced it.

Reply:
Also, done